test/test_kinetic_cafe_error.rb

```diff
@@ -13,7 +13,7 @@
       end
     end
 
-    describe '#messsage and #i18n_message' do
+    describe '#message and #i18n_message' do
```

Comment:
Is there any reason you're testing both at once? It looks like you're only testing `#i18n_message`